lib/rules/selector-max-specificity/__tests__/index.mjs

```diff
@@ -260,47 +260,47 @@ testRule({
 		},
 		{
 			code: '.cd { .de { .fg {} } }',
-			message: messages.expected('.cd .de .fg', '0,2,1'),
+			message: messages.expected('.fg', '0,2,1'),
 			line: 1,
 			column: 13,
 			endLine: 1,
 			endColumn: 16,
 		},
 		{
```

Comment:
minor

Can you add `.abc { & & {} }`?
i.e. `.abc .abc .abc {}` fails too

Reply:
We don't try to determine from which part of a selector the limit was exceeded. We always compare the final specificity of a complete selector agains the user config.

So in your example the message would contain `& &`.